homeassistant/components/switch/broadlink.py

```diff
@@ -21,7 +21,10 @@
                                  CONF_TYPE)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['broadlink==0.2']
+
+REQUIREMENTS = ['https://github.com/mjg59/python-broadlink/archive'
```

Comment:
I am not sure I understand what you want here. But each Broadlink component will have a own ip address